settings.gradle

```diff
@@ -19,3 +19,6 @@ include ':usecaseImpl'
 include ':dataSourceDI'
 include ':repositoryDI'
 include ':api'
+include ':playFeature'
+include ':settingsFeature'
+include ':leaderboardFeature'
```

Comment:
why do you need these dependencies here, use case should 'know' nothing about presentation